test/GPv2AccessControl.test.ts

```diff
@@ -0,0 +1,89 @@
+import { expect } from "chai";
```

Comment:
I just noticed now, but I think this file should have been renamed to `GPv2AllowListAuthentication.test.ts`.

Reply:
Yes sorry. I will update this ASAP. Possibly in one of the next related tasks.